create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -93,6 +98,16 @@ void shouldExtractDoiAndPlaceItInsideReferenceObject() throws IOException {
         assertThat(createPublicationRequest.getEntityDescription().getReference().getDoi(), equalToObject(expectedURI));
     }
 
+    @Test
+    void shouldExtractMainTitleAndPlaceItInsideEntityDescriptionObject() throws IOException {
+        var scopusFile = IoUtils.stringFromResources(Path.of(SCOPUS_XML_0000833530));
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+        S3Event s3Event = createS3Event(uri);
+        CreatePublicationRequest createPublicationRequest = scopusHandler.handleRequest(s3Event, CONTEXT);
+        String actualMainTitle = createPublicationRequest.getEntityDescription().getMainTitle();
+        assertEquals(HARDCODED_EXPECTED_TITLE_IN_0000833530, actualMainTitle);
```

Comment:
This test does not test only the placement of the title. It also tests that XML elements are serialized properly. This should be another test.

Reply:
There should be fresher changes than this. (amongst other this should have been fixed)